internal/resources/metal/organizations/datasource.go

```diff
@@ -0,0 +1,86 @@
+package organizations
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/equinix/equinix-sdk-go/services/metalv1"
```

Comment:
We should keep packngo in place and only migrate the resource from SDK to framework.  Changing the SDK will probably be too much change for one PR.